holidayextras/ui-toolkit
# UXUI-205 Adding docs for tile

#### What does this PR do?
Adds a basic tile component that accepts an `image`, `title` and optional children.

#### What unit or integration tests does this PR have?
N/A

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Check that the code seems sane and that the documentation describes sufficiently how to use it.
`grunt docs` will build the docs for you to see the example etc.

#### How should this be manually tested?
Run `grunt docs` & check the example [http://localhost:4000](http://localhost:4000)

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
[UXU-205](https://holidayextras.jira.com/browse/UXUI-205)

#### Screenshots (if appropriate)
![tile](https://cldup.com/hq3GPvg7vr.png)

#### What gif best describes how you feel about this work?
![tile_gif](http://media.giphy.com/media/zeIxpzLsCBAmA/giphy.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
